# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../guppi/compile-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur Guppi-0.40.3-orig/src/libguppispecfns/const.c Guppi-0.40.3/src/libguppispecfns/const.c
--- Guppi-0.40.3-orig/src/libguppispecfns/const.c	2009-08-24 00:51:18.000000000 +0000
+++ Guppi-0.40.3/src/libguppispecfns/const.c	2009-08-24 00:52:27.000000000 +0000
@@ -93,11 +93,13 @@
 double LOGSQ2 = 3.46573590279972654709E-1;	/* log(2)/2 */
 double THPIO4 = 2.35619449019234492885;	/* 3*pi/4 */
 double TWOOPI = 6.36619772367581343075535E-1;	/* 2/pi */
+#ifndef INFINITY
 #ifdef INFINITIES
 double INFINITY = 1.0 / 0.0;	/* 99e999; */
 #else
 double INFINITY = 1.79769313486231570815E308;	/* 2**1024*(1-MACHEP) */
 #endif
+#endif
 
 #ifdef MINUSZERO
 double NEGZERO = -0.0;
